apps/ensadmin-next/src/components/app-sidebar.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { Activity, DatabaseIcon } from "lucide-react";
+import * as React from "react";
+
+import { NavMain } from "@/components/nav-main";
+
+import { Sidebar, SidebarContent, SidebarHeader, SidebarRail } from "@/components/ui/sidebar";
+import { ConnectionSelector } from "./connections/connection-selector";
+
+const navItems = [
+  {
+    title: "Status",
+    url: "/status",
+    icon: Activity,
+  },
+  {
+    title: "APIs",
+    url: "#",
+    icon: DatabaseIcon,
+    isActive: true,
+    items: [
+      {
+        title: "GraphQL",
+        url: "/gql/ponder",
+      },
+      {
+        title: "Subgraph",
```

Comment:
```suggestion
        title: "GraphQL (Subgraph-style)",
```

Trying to find a way to communicate / differentiate the two GraphQL endpoints. Shared my suggestion in 2 comments here.